src/index.html

```diff
@@ -6,7 +6,314 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
+        </a>
+      </div>
+      <nav class="navbar">
+        <ul class="navbar__list">
+          <li class="navbar__item">
+            <a href="/apple" class="navbar__link is-active">Apple</a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/samsung" class="navbar__link">Samsung</a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/smartphones" class="navbar__link">Smartphones</a>
+          </li>
+
+          <li class="navbar__item">
+            <a
+              href="/laptops+computers"
+              class="navbar__link"
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/gadgets" class="navbar__link">gadgets</a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/tablets" class="navbar__link">Tablets</a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/photo" class="navbar__link">Photo</a>
+          </li>
+
+          <li class="navbar__item">
+            <a href="/video" class="navbar__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products page__main">
+      <div class="card" data-qa="card">
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/0f274b64-5ed6-4f76-88d6-2460d286827e)
Article\section would be better